openpype/hosts/fusion/plugins/publish/render_local.py

```diff
@@ -17,18 +17,20 @@ class Fusionlocal(pyblish.api.InstancePlugin):
     families = ["render.local"]
 
     def process(self, instance):
-
-        # This plug-in runs only once and thus assumes all instances
-        # currently will render the same frame range
         context = instance.context
-        key = f"__hasRun{self.__class__.__name__}"
-        if context.data.get(key, False):
-            return
-
-        context.data[key] = True
 
+        # Start render
         self.render_once(context)
 
+        # Log render status
+        self.log.info(
+            "Rendered '{nm}' for asset '{ast}' under the task '{tsk}'".format(
+                nm=instance.data.name,
+                ast=instance.data.asset,
+                tsk=instance.data.task,
```

Comment:
This doesn't work, right? It should be `instance.data["name"]`, `instance.data["asset"]`, etc. Right? :)
Or is this some legacy backwards compatible way of what Pyblish offers? Have never seen this.

@jakubjezek001 is this the kind of logic you'd have expected? Note that technically the second instance logging it didn't actually _just render it_ but the previous instance had already rendered the comp. But at least this shows a log per instance.

Reply:
No you're correct. idk how I could do that. Fixed it!